readme.md

```diff
@@ -1,3 +1,5 @@
+[![Coverage Status](https://coveralls.io/repos/github/Sviridy/laba/badge.svg)](https://coveralls.io/github/Sviridy/laba)
```

Comment:
Wrong badge configuration, please fix it.

Reply:
fixed